src/main/java/com/crud/kodillalibrary/copy/CopyController.java

```diff
@@ -0,0 +1,39 @@
+package com.crud.kodillalibrary.copy;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/library/copies")
+public class CopyController {
+    private final CopyMapper copyMapper;
+    private final CopyService copyService;
+
+    @Autowired
+    public CopyController(CopyMapper copyMapper, CopyService copyService) {
+        this.copyMapper = copyMapper;
+        this.copyService = copyService;
+    }
+
+    @PostMapping
+    Long addCopy(@RequestBody CopyDto copyDto) {
+        Copy copy = copyService.addCopy(copyMapper.mapToCopy(copyDto));
+        return copy.getId();
+    }
+
+    @PatchMapping("{id}")
+    CopyDto changeStatus(@RequestBody CopyDto copyDto, @PathVariable long id) {
+        return copyMapper.mapToCopyDto(copyService.changeStatus(copyDto, id));
```

Comment:
jeśli ta metoda zmienia tylko status, to powinna przyjmowac Status jako pathvariable
```
    @PatchMapping("{copyId}/{status}")
    CopyDto changeStatus(@PathVariable long id, @PathVariable Status status) {
```